api/migrations/20190306102126-user.js

```diff
@@ -14,8 +14,28 @@ exports.setup = function(options, seedLink) {
   seed = seedLink;
 };
 
-exports.up = function(db) {
-  return null;
+exports.up = function(db, callback) {
```

Comment:
This looks good but next time you can do all your inserts in one command:

https://stackoverflow.com/questions/5526917/how-to-do-a-batch-insert-in-mysql